x-pack/plugins/reporting/public/panel_actions/get_csv_panel_action.tsx

```diff
@@ -73,7 +74,7 @@ export class ReportingCsvPanelAction implements ActionDefinition<ActionContext>
   }
 
   public async getSearchSource(savedSearch: SavedSearch, _embeddable: ISearchEmbeddable) {
-    const [{ uiSettings }, { data }] = await this.startServices$.pipe(first()).toPromise();
+    const [{ uiSettings }, { data }] = await firstValueFrom(this.startServices$.pipe(first()));
```

Comment:
nit: maybe `first()` is not necessary here:

```suggestion
    const [{ uiSettings }, { data }] = await firstValueFrom(this.startServices$);
```